content/en/docs/reference/node/kubelet-files.md

```diff
@@ -0,0 +1,158 @@
+{{<glossary_tooltip text="container runtime" term_id="container-runtime">}} to retrieve
+the current state of containers. So long as you provide a _kubeconfig_ (API client configuration)
+to the kubelet, the kubelet does connect to your control plane; otherwise the node operates in
+_standalone mode_.
+
+On Linux nodes, the kubelet also relies on reading cgroups and various system files to collect metrics.
+
+On Windows nodes, the kubelet collects metrics via a different mechanism that does not rely on
+paths.
+
+There are also a few other files that are used by the kubelet as well as kubelet communicates using local Unix-domain sockets. Some are sockets that the
+kubelet listens on, and for other sockets the kubelet discovers them and then connects
+as a client.
+
+{{< note >}}
+
+This page lists paths as Linux paths, which map to the Windows paths by adding a root disk
+`C:\` in place of `/` (unless specified otherwise). For example, `/var/lib/kubelet/device-plugins` maps to `C:\var\lib\kubelet\device-plugins`.
+
+{{< /note >}}
+
+## Configuration
+
+### Kubelet configuration files
+
+The path to the kubelet configuration file can be configured
+using the command line argument `--config`. kubelet also supports
+[drop-in configuration files](/docs/tasks/administer-cluster/kubelet-config-file/#kubelet-conf-d)
+to enhance configuration.
+
+### Certificates
+
+Certificates and private keys are typically located at `/var/lib/kubelet/pki`,
+but can be configured using the `--cert-dir` kubelet command line argument.
+Names of certificate files are also configurable.
+
+### Manifests
+
+Manifests for static pods are typically located in `/etc/kubernetes/manifests`.
+Location can be configured using the `staticPodPath` kubelet configuration option.
+
+### Systemd unit settings
+
+When kubelet is running as a systemd unit, some kubelet configuration may be declared
+in systemd unit settings file. Typically it includes:
+
+- command line arguments to [run kubelet](/docs/reference/command-line-tools-reference/kubelet/)
+- environment variables, used by kubelet or [configuring golang runtime](https://pkg.go.dev/runtime#hdr-Environment_Variables)
+
+## State
+
+### Checkpoint files for resource managers {#resource-managers-state}
+
+All resource managers keep the mapping of Pods to allocated resources in state files.
+State files are located in the kubelet's base directory, also termed the _root directory_
+(but not the same as `/`, the node root directory). You can configure the base directory
+for the kubelet
+using the kubelet command line argument `--root-dir`.
+
+Names of files:
+
+- `memory_manager_state` for the [Memory Manager](/docs/tasks/administer-cluster/memory-manager/)
+- `cpu_manager_state` for the [CPU Manager](/docs/tasks/administer-cluster/cpu-management-policies/)
+- `dra_manager_state` for [DRA](/docs/concepts/scheduling-eviction/dynamic-resource-allocation/)
+
+### Checkpoint file for device manager {#device-manager-state}
```

Comment:
It's still not very clear to me. Isn't device manager also a resource manager?

Dividing state checkpoint files into 3 categories looks a bit artificial to me. I'd prefer to see them together, or divided in 2 categories by the directory where they store their checkpoints.

How about this way of representing them?

|Directory|File|Manager|Note|
|---------|----|---------|----|
|Kubelet root directory, /var/lib/kubelet by default|`memory_manager_state`|[Memory Manager](/docs/tasks/administer-cluster/memory-manager/)||
||`cpu_manager_state`|[CPU Manager](/docs/tasks/administer-cluster/cpu-management-policies/)||
||`dra_manager_state`|[DRA Manager](/docs/concepts/scheduling-eviction/dynamic-resource-allocation/)||
||`pod_manager_state`|Pod Manager|with [in-place Pod vertical scaling](https://github.com/docs/concepts/workloads/autoscaling/#in-place-resizing) enabled|
|/var/lib/kubelet/device-plugins|`kubelet_internal_checkpoint`|[Device Manager](/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/#device-plugin-integration-with-the-topology-manager)||

Reply:
I don't think Markdown is good at tables, and I don't believe embedding an HTML table into markdown is a good idea in this case.
The files/directories can be enumerated with bulletins which are clear enough and easy to maintain as website source.

Reply:
I split it separately as it is placed in a differnet location. I can merge back if you feel strongly about it